src/components/card/CardSection.js

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {BaseComponent} from '../../commons';
+import View from '../view';
+import Text from '../text';
+import Image from '../image';
+import asCardChild from './asCardChild';
+
+/**
+ * @description: Card.Image, part of the Card component belongs inside a Card (better be a direct child)
+ * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/CardsScreen.js
+ */
+class CardSection extends BaseComponent {
+  static displayName = 'Card.Section';
+
+  static propTypes = {
+    /**
+     * Text content for the CardSection.
+     * Example: content={[{text: 'You’re Invited!', props: {text70: true, dark10: true}}]}
+     */
+    content: PropTypes.arrayOf(PropTypes.shape({
+      text: PropTypes.string,
+      props: Text.propTypes
```

Comment:
Do you think that maybe we can flat this shape and allow the user to pass `text` and directly other text props? 

so instead of passing 
```
content: {text: 'title', props: {numberOfLines: 2, color: 'red'}}
```
the user can pass 
```
content: {text: 'title', numberOfLines: 2, color: 'red'}
```

Reply:
Done